source/guide/index.md

```diff
@@ -10,124 +10,125 @@ version: 2.1
+The structure of Weex is decoupled: the render engines are separated from the syntax layer. Weex does not rely on any specific front-end framework; however, Weex mainly supports [Vue.js](https://vuejs.org/) and [Rax](https://alibaba.github.io/rax/).
 
-Another primary goal of Weex is to keep up with modern development technologies and platform capabilities both for web and native. Productivity and performance can coexist in Weex. Writing Weex pages feels the same like writing web pages. Rendering Weex pages is just the same as rendering native pages.
+Another primary goal of Weex is to keep up with modern development technologies and platform capabilities both for web and native. Productivity and performance can coexist in Weex. Writing Weex pages feels the same as writing web pages. In fact, rendering Weex pages is the same as rendering native pages.
 
 ## Overview
 
-If you just want to try Weex, You don't need to install anything. There is an [online playground](http://dotwe.org/vue/) of Weex, you can write single page examples on it without any configurations. The source code should be written in Vue.js [single file component](https://vuejs.org/v2/guide/single-file-components.html) syntax (also known as the `*.vue` files), and the render result of the web platform will be displayed in a mocked phone shell.
+If you just want to try Weex, you do not need to install anything. There is an [online playground](http://dotwe.org/vue/) for Weex wherein you can write single page examples without any installation or configuration. The source code should be written in Vue.js [single file component](https://vuejs.org/v2/guide/single-file-components.html) syntax (also known as the `*.vue` files), and the rendered result from editor pane will be displayed in a mock phone shell.
 
 Here is an [example](http://dotwe.org/vue/8da01827631b21150a12dd54d7114380) written in Weex and Vue.js:
 
 ![Weex Example](./images/weex-example-yo.png)
 
-This example renders a word "Yo" in the center of the screen. If you want to preview the rendered result on mobile devices, you need to install the [Weex playground app](/tools/playground.html) or integrate Weex SDK into your own app. Then scan your page's QR code with your playground app or your own app (with the QR scan feature to load scanned URL with Weex SDK), and you will see the rendered result on your device.
+This example renders a word "Yo" in the center of the screen. If you want to preview the rendered result on mobile devices, you need to install the [Weex playground app](/tools/playground.html), or integrate Weex SDK into your own app and scan your page's QR code with your playground app or your own app (with the QR scan feature to load scanned URL with Weex SDK). Thereafter, you will see the rendered result on your device.
```

Comment:
We can change "...rendered result on mobile devices, you need...." to "...rendered result on a mobile device, you need..."